react/features/toolbox/components/web/Toolbox.tsx

```diff
@@ -348,9 +348,10 @@ const styles = () => {
         contextMenu: {
             position: 'relative' as const,
             right: 'auto',
-            maxHeight: 'inherit',
             margin: 0,
-            marginBottom: '8px'
+            marginBottom: '8px',
+            maxHeight: "60vh",
+            overflow: "scroll",
```

Comment:
We don't want the menu to scroll all the time. The context menu already has `overflow-y: auto` so this isn't needed